ansible/README.md

```diff
@@ -118,12 +118,13 @@ Transpiling the map styles from CartoCSS to Mapnik XML does not require database
 expects the XML files at /root/packages.
 
 ```sh
-git clone https://github.com/OpenRailwayMap/OpenRailwayMap-CartoCSS
+git clone https://github.com/OpenRailwayMap/OpenRailwayMap-CartoCSS.git
 cd OpenRailwayMap-CartoCSS
-carto project.mm > project.xml
-carto maxspeed.mm > maxspeed.xml
-carto signals.mm > signals.xml
-cp project.xml maxspeed.xml signals.xml /root/packages`
+for style in standard maxspeed signals electrification gauge
```

Comment:
This changes "project" to "standard". I think this is correct, I just want to make sure. What about just globbing all mm files so this does not need any more updates?

Reply:
done.

Reply:
This called carto on the .mm file, and now it does that on the .mml file. Is that correct?

Reply:
Yes, `.mm` was indeed a typo.